src/Controller/CRUDController.php

```diff
@@ -1365,6 +1375,14 @@ protected function getAclUsers()
         }
 
         return \is_array($aclUsers) ? new \ArrayIterator($aclUsers) : $aclUsers;
+
+//        if (!$this->has('sonata.admin.security.acl_user_manager')) {
+//            return new \ArrayIterator([]);
+//        }
+//
+//        $aclUsers = $this->get('sonata.admin.security.acl_user_manager')->findUsers();
+//
+//        return \is_array($aclUsers) ? new \ArrayIterator($aclUsers) : $aclUsers;
```

Comment:
```suggestion
        return $aclUsers;

//        if (!$this->has('sonata.admin.security.acl_user_manager')) {
//            return [];
//        }
//
//        $aclUsers = $this->get('sonata.admin.security.acl_user_manager')->findUsers();
//
//        return $aclUsers;
```

ArrayIterator is not needed to me

Reply:
I just wanted to introduce the interface and deprecate not using it, changing the type declaration of `CRUDController::getAclUsers` (now it is \Traversable) can be done in another PR because there are some other changes implied, like changing `CRUDController::getAclRoles` for consistency and `AdminObjectAclData` constructor that IMO are not related to this PR.

Reply:
Didn't see the phpdoc.